src/pages/otp/otp.html

```diff
@@ -0,0 +1,34 @@
+
+<ion-content padding>
+  <div class="vertical-container">
+    <div class="vertical-align">
+      <div *ngIf="showRequest">
+        <form [formGroup]="request">
+          <h2>Please enter your phone number.</h2>
+          <p>Including the international number digit(s).</p>
```

Comment:
Does not follow proper conventions. This should be:
form [formGroup]="request" (ngSubmit)="requestCode()" novalidate